tests/integration/test_camera_picamera2.py

```diff
@@ -1,13 +1,12 @@
 """
-Test Picamera2 camera physically.
+Test Picamera2 camera physically and verifies configuration.
 """
 
 import pathlib
 
 import cv2
 
-from modules.camera import camera_factory
-
+from modules.camera import camera_factory, camera_config
```

Comment:
Same here: separate into 2 lines and make sure 2 spaces between imports and global constants